submissions/andysmokk/dom-js/sass/variables/_variables.scss

```diff
@@ -0,0 +1,26 @@
+:root {
+    --default-font: 'Lemonada';
+    --main-gray-color: #b1a6a6;
```

Comment:
Remove `color` postfix from colour's names.
When you name your style vars try to select a name by "What is it?" not "Where will it be used?"

Reply:
Fixed.